### DISCUSSION TITLE
Import mesh to a certain position

### DISCUSSION THREAD

**Discussion by flaviu22** [cnr-isti-vclab/meshlab#868] [Q&A] [answered]
I want to ask you: can I import a mesh file (as STL) to a certain position (not in the center) ? If is there possible, can you give me small references to complete that ?

Kindly thank you.

**Comment by alemuntoni** [comment, marked as answer]
What do you mean with "not in the center"? 
Every mesh has coordinates and the mesh appears "centered" just for visualization.
If you need to visualize where the mesh is spatially placed, you can use the "draw XYZ axes" tool.

**Comment by flaviu22** [comment]
Thank you. I get it.
